File: pset7/cash/cash.py
from cs50 import get_float

x = -1
while x < 0:
    x = get_float("Change owed: ")

coins = x // 0.25
x = round(x % 0.25,2)


coins += x //0.1
x = round(x % 0.1,2)


coins += x//0.05
x = round(x % 0.05,2)

coins += x // 0.01

print(int(coins))
# there's some iffy rounding in the script

File: pset1/cash/cash.c
#include <stdio.h>
#include <cs50.h>
#include <math.h>

int main(void)
{
    float f;
    int c, r; //coins, remainder,

    do
    {
        f = get_float("Change owed: ");
    }
    while (f < 0);

    f = f * 100;
    r = round(f);

    c = r / 25; //Quarters
    r = r % 25;

    c = c + r / 10; //Dimes
    r = r % 10;

    c = c + r / 5; //Nickes
    r = r % 5;

    c = c + r; //Pennies

    printf("%i\n", c);
}